From: Jan Gerber <j@xiph.org>
Date: Fri, 16 Aug 2013 08:40:53 +0000 (+0200)
Subject: print error instead of printing uninitialized memory to terminal if no input is specified
X-Git-Url: http://git.xiph.org/?p=ffmpeg2theora.git;a=commitdiff_plain;h=d462b50fa8d9462b847a4e574b2d50fc4d191352

print error instead of printing uninitialized memory to terminal if no input is specified
---

diff --git a/src/ffmpeg2theora.c b/src/ffmpeg2theora.c
index 8bebf28..410d502 100644
--- a/src/ffmpeg2theora.c
+++ b/src/ffmpeg2theora.c
@@ -2773,6 +2773,9 @@ int main(int argc, char **argv) {
             outputfile_set=1;
         }
         optind++;
+    } else {
+        fprintf(stderr, "ERROR: no input specified\n");
+        exit(1);
     }
     if(optind<argc) {
         fprintf(stderr, "WARNING: Only one input file supported, others will be ignored\n");
